src/CoreWCF.Primitives/src/CoreWCF/Security/MessageCredentialType.cs

```diff
@@ -12,9 +12,9 @@ public enum MessageCredentialType
         IssuedToken
     }
 
-    internal static class MessageCredentialTypeHelper
+    public static class MessageCredentialTypeHelper
     {
```

Comment:
Helper class should be internal

Reply:
made internal... move MessageSecurityOverNetTCP to Primitive

Reply:
No, MessageSecurityOverNetTcp belongs in .NetTcp package. Duplicate the helper if needed, but leave NetTcp specific types in the correct package.

---
In reply to: [599847541](https://github.com/CoreWCF/CoreWCF/pull/331#discussion_r599847541) [](ancestors = 599847541)